lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -24,21 +24,24 @@ defmodule MembraneLiveWeb.EventChannel do
 
       {:ok, true} ->
         :ets.insert_new(:presenters, {id, MapSet.new()})
+        :ets.insert_new(:presenting_requests, {id, MapSet.new()})
 
         with {:ok, %{"user_id" => uuid}} <- Tokens.auth_decode(token),
              {:ok, name} <- Accounts.get_username(uuid),
              {:ok, email} <- Accounts.get_email(uuid),
              [] <- Presence.get_by_key(socket, email),
              {:ok, socket} <- create_event_stream(id, socket),
              {:ok, is_presenter} = check_if_presenter(email, reloaded, id),
+             {:ok, is_request_presenting} = check_if_request_presenting(email, reloaded, id),
              is_moderator <- Webinars.check_is_user_moderator(uuid, id) do
           {:ok, socket} = if is_presenter, do: join_event_stream(socket), else: {:ok, socket}
 
           {:ok, _ref} =
             Presence.track(socket, email, %{
               name: name,
               is_moderator: is_moderator,
-              is_presenter: is_presenter
+              is_presenter: is_presenter,
+              request_presenting: is_request_presenting
```

Comment:
wouldn't it be better to stick to the convention?

```suggestion
              is_request_presenting: is_request_presenting
```